Source/WebCore/dom/ScriptExecutionContext.h

```diff
@@ -270,6 +270,7 @@ class ScriptExecutionContext : public SecurityContext, public TimerAlignment {
 
     virtual JSC::VM& vm() = 0;
     virtual Ref<JSC::VM> protectedVM();
+    virtual JSC::VM* existingVMOrNull() = 0;
```

Comment:
I think the established convention for "getter that returns nullable reference to object that may not exist" would be `vmIfExists()`:

```
virtual JSC::VM* vmIfExists() const = 0;
```

(This should also be made `const` to reflect that it shouldn't instantiate a VM as a result of calling it)